components/atom/tag/src/Actionable/index.js

```diff
@@ -54,7 +55,8 @@ ActionableTag.propTypes = {
   onClick: PropTypes.func,
   target: PropTypes.oneOf(['_self', '_blank', '_parent', '_top']),
   rel: PropTypes.arrayOf(PropTypes.oneOf(Object.values(LINK_TYPES))),
-  linkFactory: PropTypes.func
+  linkFactory: PropTypes.func,
+  value: PropTypes.oneOfType([PropTypes.string, PropTypes.number])
```

Comment:
That could be anything, maybe someone needs to store and object instead of just a string or number.

Reply:
i see this similar to an optionList component where the value must be a string, i added number to add some other scenarios but i dont know if we should store objects as value in this type of components

Reply:
Interesting. Maybe when its needed would not break anything and we will iterate it.